spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -49,6 +49,9 @@
 from spyder.utils.conda import get_list_conda_envs
 
 
+PY312_OR_GREATER = sys.version_info[:2] >= (3, 12)
```

Comment:
Please move this comment to the `spyder.plugins.ipythonconsole.tests.conftest` module and import it from there.